LEAF_Request_Portal/templates/view_session_expire.tpl

```diff
@@ -0,0 +1,33 @@
+{strip}<!DOCTYPE html>
+<html lang="en">
+<head>
+    <title>{$title|sanitize}</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <style type="text/css" media="screen">
+        @import "css/style.css";
+    </style>
+    <style type="text/css" media="print">
+        @import "css/printer.css";
+    </style>
+    <div id="header">
+        <div style="cursor: pointer" onclick="window.location='./'">
+            <span style="position: absolute"><img src="images/VA_icon_small.png" style="width: 80px" alt="VA logo" /></span>
+            <span id="headerDescription">{$title|sanitize}</span>
+        </div>
+        <span id="headerTab"></span>
+        <span id="headerTabImg"><img src="images/tab.png" alt="tab" /></span>
+    </div>
+    <div id="body">
+        <div id="content">
+            <div id="bodyarea">
+                <div class="card" style="width: 50%; margin: auto; padding: 16px">
+                    <h2>Your Session Has Expired</h2>
+                    <p>Return to your <a href="{$previousPage}">LEAF site</a></p>
```

Comment:
Should this route to the "/login" page?  I don't think so, but probably ought to be asked.

Reply:
This has been changed in the latest commit as not really need for the previous page if next page should be login anyway, not back to the old page.